index.html

```diff
@@ -12,10 +12,115 @@
+        </section> 
+        
+        <nav>
+            <section class="big-screen">
+                <ul>
+                    <li><a href="#">Meet Karma</a></li>
+                    <li><a href="#">How It Works</a></li>
+                    <li><a href="#">Store</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">Help</a></li>
+                    <li><a href="#">Login</a></li>
+                </ul>
+            </section>
+            <section class="mobile-screen" >
+                <div id="start-link">
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                    <img src="./img/menu-hamburger.svg">
+                    </a>
+                </div>
+              
+                <div class="topnav" id="myLinks">
+                    <ul>
+                        <li> <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                            <img src="./img/menu-hamburger.svg">
+                            </a></li>
+                       
+                        <li><a href="#">Meet Karma</a></li>
+                        <li><a href="#">How It Works</a></li>
+                        <li><a href="#">Store</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><a href="#">Help</a></li>
+                        <li><a href="#">Login</a></li>
+                    </ul>
+                </div>
+                
+            </section>
+        </nav>
+        
+    </header>
+    <!-- my main section -->
+    <main>
+        <section class="hero">
+            <div class="hero-headings">
+                <p class="hero-headings-bigsize">Introducing Karma</p>
+                <p class="hero-headings-smallsize">Bring WIFI with you,everywhere you go.</p>
+            </div>
+            <div class="hero-button">
+                <button>Learn More</button>
+            </div>
+            
+        </section>
+        <section class="container">
+            <section class="container-heading">
+                <p>Everyone needs a little Karma</p>
+            </section>
+            <section class="container-icons">
+                <section class="container-icons-wrapper">
+                    <img src="./img/icon-devices.svg" alt="picture showing a computer">
+                    <p>Internet for all devices</p>
+                </section>   
+                <section class="container-icons-wrapper">
+                    <img src="./img/icon-coffee.svg" alt="picture showing a computer">
+                    <p>Boost your productivity</p>
+                </section>   
+                <section class="container-icons-wrapper">
+                    <img src="./img/icon-refill.svg" alt="picture showing a computer">
+                    <p>Pay as you go</p>
+                </section>       
+            </section>
+        </section>
+        <hr>
```

Comment:
Can you add some styling to the `hr` so that it doesn't span across the whole width of the screen?

Reply:
Thank you very much, I changed its' width to 50%, looks much better now.
#main-hr{
    width:50%;
}